pyproject.toml

```diff
@@ -34,7 +34,7 @@ dev = [
 clinvar-ingest = "clinvar_ingest.main:main"
 
 [tool.setuptools.packages.find]
-include = ["clinvar_ingest"]
+include = ["clinvar_ingest*"]
```

Comment:
Nested package (`clinvar_ingest.cloud`) wasn't getting packaged up in the `pip install .` build.